package.json

```diff
@@ -26,11 +26,12 @@
     "backstop-test": "npm run backstop-remote & just ember test; npm run backstop-stop"
   },
   "dependencies": {
+    "backstopjs": "^4.4.2",
```

Comment:
Copying comment from the blueprint here because I want to be super sure about this...

> 
> Again, just checking -- did you test that this works when installing on a plain ember app? E.G. by adding to https://github.com/ember-learn/super-rentals
> 
> Do manual CLI commands run from root work? e.g. using ember backstop-approve or what about npm run backstop-approve ?
> 
> There is a lot of path magic with ember -- so it would be nice to check that utility functions still work if we move this reference.
> 
>